files/qualys_reporting

```diff
@@ -0,0 +1,4 @@
+Defaults:qualys !mail_always
+Defaults:qualys !requiretty
+%qualys ALL=(ALL) NOPASSWD: NOMAIL: ALL
+qualys ALL=(ALL) NOPASSWD: NOMAIL: ALL
```

Comment:
I suggest we move this to hiera data so that it could easily be customized if needed by a specific project's control repo.

Reply:
I was putting that in a file since I wasn't really expecting that config to ever really change that much per cluster. But yeah there will probably be some reason eventually so I can put that in hiera. For unsetting it, I'll probably go with a boolean to enable/disable qualys sudo, since that reads a little better than setting this hiera data to empty.

Reply:
This has been done.